packages/runtime/container-runtime/src/test/garbageCollection.spec.ts

```diff
@@ -513,6 +515,40 @@ describe("Garbage Collection Tests", () => {
         });
     });
 
+    describe("GC run count", () => {
+        const gcEndEvent = "GarbageCollector:GarbageCollection_end";
+
+        it("increments GC run count in logged events correctly", async () => {
+            const garbageCollector = createGarbageCollector();
+
+            await garbageCollector.collectGarbage({});
+            assert(
+                mockLogger.matchEvents([{ eventName: gcEndEvent, gcRunCount: 0 }]),
```

Comment:
Now that I think about how it'll show up in telemetry, I wonder if we should call this `previousGCRunCount` because `gcRunCount` makes me think we are on run number x.

Reply:
`count` here tells how many times GC has run when this event was logged. All logs generated before and during the first run has this as 0 because GC has not completely run. Maybe we can call this `completedGCRuns` to better reflect this behavior.

Reply:
Looks like you updated the code but not the tests yet?